sys-libs/libatomic_ops/libatomic_ops-7.6.2.recipe

```diff
@@ -0,0 +1,78 @@
+SUMMARY="An atomic memory update operations portable implementation"
+DESCRIPTION="Semi-portable access to hardware-provided atomic memory update \
+operations on a number architectures."
+HOMEPAGE="https://github.com/ivmai/libatomic_ops"
+COPYRIGHT="1991-1994 by Xerox Corporation.  All rights reserved.
+	1996-1999 by Silicon Graphics.  All rights reserved.
+	2003-2011 Hewlett-Packard Development Company, L.P.
+	2005,2007 Thiemo Seufer
+	2007 by NEC LE-IT
```

Comment:
We usually drop the "All rights reserved", although I don't know if we should continue doing so.

```diff
-COPYRIGHT="1991-1994 by Xerox Corporation.  All rights reserved.
+COPYRIGHT="1991-1994 by Xerox Corporation
-	1996-1999 by Silicon Graphics.  All rights reserved.
+	1996-1999 by Silicon Graphics
 	2003-2011 Hewlett-Packard Development Company, L.P.
 	2005,2007 Thiemo Seufer
-	2007 by NEC LE-IT
+	2007 NEC LE-IT
 	2016 Marek Vasut"
```